packages/chakra-ui/src/UpDownWidget/UpDownWidget.tsx

```diff
@@ -66,7 +66,7 @@ export default function UpDownWidget<
         <FormLabel htmlFor={id}>{label || schema.title}</FormLabel>
       ) : null}
       <NumberInput
-        value={value ?? ""}
+        value={value || ""}
```

Comment:
Is value always a string here? `0 || "" === ""` but `0 ?? "" === 0`

Reply:
Hmmm, good catch.

Reply:
Reverted